manifests/t/Tencent/WeCom/4.0.0.6023/Tencent.WeCom.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Tencent.WeCom
+PackageVersion: 4.0.0.6023
+PackageLocale: en-US
+Publisher: Tencent

```

```suggestion
Publisher: 腾讯科技(深圳)有限公司
```
This must match Control Panel.